### PR TITLE
Default to templated value of displayField 

### PR DESCRIPTION
**Pull request by rgwozdz**
Default to templated value of displayField if no metadata value and fields array is empty.

### CODE REVIEW

File: src/templates.js [hunk 1]
```diff
@@ -53,7 +53,7 @@ function renderLayer (featureCollection = {}, options = {}) {
   if (json.drawingInfo) json.drawingInfo.renderer = renderers[json.geometryType]
   if (json.timeInfo) json.timeInfo = metadata.timeInfo
   if (json.maxRecordCount) json.maxRecordCount = metadata.maxRecordCount || 2000
-  if (json.displayField) json.displayField = metadata.displayField || json.fields[0].name
+  if (json.displayField) json.displayField = metadata.displayField || json.fields[0].name || json.displayField
```
Comment (dmfenton) [inline src/templates.js]: this will still error out if json.fields is null

Comment (rgwozdz) [reply src/templates.js]: Now updated with `_.get(json, 'fields[0].name')`

Comment (dmfenton) [reply src/templates.js]: lodash ftw, let's just add a test here.

Comment (rgwozdz) [reply src/templates.js]: Added tests for `fields: null` and `fields: []`.

**Pull request merged**